app/src/main/java/com/mapbox/maps/testapp/examples/DebugModeActivity.kt

```diff
@@ -165,31 +168,20 @@ class DebugModeActivity : AppCompatActivity() {
     }
   }
 
+  @SuppressLint("SetTextI18n")
   override fun onStart() {
     super.onStart()
-    mapView.onStart()
-    mapView.setOnFpsChangedListener {
+    binding.mapView.setOnFpsChangedListener {
       runOnUiThread {
-        fpsView.text = "${it.toInt()} FPS"
+        binding.fpsView.text = "${it.toInt()} FPS"
```

Comment:
We can use string resources to remove `@SuppressLint("SetTextI18n")`.

Reply:
Yep!